### PR TITLE
[HOTFIX][#2821] Fix for broken Group page functionality

### PR DESCRIPTION
**Pull request by Maurier**
https://github.com/hydroshare/hydroshare/issues/2821 This PR addresses a lot of issues created before last release and that seem to have made it to www due to lack of testing. In order to fix these I had to revert changes from previous conflicting PRs. I then fixed the things those PRs were addressing in a clean way.

See https://github.com/hydroshare/hydroshare/issues/2821#issuecomment-501079074 for the list of bugs this is addressing.

Here is a traceback for what seems to have caused these issues:
- Shared By filter broken with the removal of Author data the script needs here: https://github.com/hydroshare/hydroshare/commit/f5ade7c1f8dc54c0316a56a4948ba477dfe2415e#diff-f5ff8acfcb7057a2f158ab46e6b9f380R646
- Issue raised regarding Shared By filter being broken: https://github.com/hydroshare/hydroshare/issues/3365
- An erroneous fix was produced and at the same time a Vue instance was introduced which broke jQuery bound events:
https://github.com/hydroshare/hydroshare/pull/3368

Notice that, since this is an urgent issue, I am targeting the master branch for a hotfix. 
I have not addressed the white space on top of the sub navigation because I am not sure if this was done intentionally. I would appreciate if someone could clarify.

Also, the styling and positioning of Send Invitation button has been fixed as pointed out at https://github.com/hydroshare/hydroshare/issues/2821
image

### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

File: theme/templates/pages/group.html [hunk 1]
```diff
@@ -860,13 +860,12 @@ <h4 class="modal-title" id="myModalLabel">Edit Group Details</h4>
...
         </div>
     {% endif %}
-    </div>
+
 {% endblock %}
 
 {% block extra_js %}
     {{ block.super }}
-    <script type="text/javascript" src="{{ STATIC_URL }}js/vue-2.6.10-dev.js"></script>
     <script type="text/javascript" charset="utf8" src="{{ STATIC_URL }}js/jquery.dataTables.js"></script>
```
Comment (cuahsimarko) [inline theme/templates/pages/group.html]: Isn't this removal what broke the author filtering? Do you need to add it back?

**Comment by cuahsimarko** [comment]
@Maurier I have no knowledge of or opinion on the styling issues @ChristinaB has raised and the subsequent ones you have noted.

File: theme/templates/pages/group.html (hunk 1 above)
Comment (Maurier) [reply theme/templates/pages/group.html]: This is not what broke Author filtering. I have described the cause in the description above.

**Comment by Maurier** [comment]
@sblack-usu I noticed that when the First Owner is an Organization it will not get listed in the table. This might be something we'll also have to fix.

**Comment by cuahsimarko** [comment]
There is a long e-mail thread and some old meetings about that.

**Comment by cuahsimarko** [comment]
I still am coming to terms with how the checkboxes work and what the defaults should be, but as long as you two are in agreement, go ahead and merge.

**Comment by sblack-usu** [comment]
I deployed the latest commits to dev-hs-1

**Comment by Maurier** [comment]
Tested the deployment and I found no issues. This has my approval as well.

**Comment by sblack-usu** [comment]
Ok, I'll go ahead and merge this and get it deployed to beta for testing.  We have been approved to push out a patch release this week.  @phuongdm - does tomorrow or Friday morning work for you to help me with the release?  We'll get something scheduled assuming it looks good on beta.  @cbcunc mentioning you, you're always helpful with the release.

**Comment by phuongdm** [comment]
@sblack-usu tomorrow works for me

**Comment by sblack-usu** [comment]
Tagged and pushed to beta

**Pull request merged**